pkg/cmd/cli/cmd/deploy.go

```diff
@@ -221,20 +221,31 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
 	// our client tools.
 	deploymentName := deployutil.LatestDeploymentNameForConfig(config)
 	deployment, err := o.kubeClient.ReplicationControllers(config.Namespace).Get(deploymentName)
-	if err == nil {
+	switch {
+	case err == nil:
 		// Reject attempts to start a concurrent deployment.
-		status := deployutil.DeploymentStatusFor(deployment)
-		if status != deployapi.DeploymentStatusComplete && status != deployapi.DeploymentStatusFailed {
+		if !deployutil.IsTerminatedDeployment(deployment) {
+			status := deployutil.DeploymentStatusFor(deployment)
 			return fmt.Errorf("#%d is already in progress (%s).\nOptionally, you can cancel this deployment using the --cancel option.", config.Status.LatestVersion, status)
 		}
-	} else {
-		if !kerrors.IsNotFound(err) {
-			return err
-		}
+	case !kerrors.IsNotFound(err):
+		return err
+	default:
+		// No latest deployment exists - we need to instantiate the config.
+	}
+
+	request := &deployapi.DeploymentRequest{
+		Name:   config.Name,
+		Latest: false,
+		Force:  true,
```

Comment:
Clients should have this set to true in order to force a new deployment to run.

Force won't work for paused deployments - we have an issue about enabling it but before we need to move the concept of one-off deployments upstream.